src/ingestion/process-event.ts

```diff
@@ -1,30 +1,466 @@
-import { DateTime } from 'luxon'
-import { PluginsServer, Properties } from 'types'
-import { UUIDT } from '../utils'
 import { PluginEvent } from '@posthog/plugin-scaffold'
```

Comment:
Compare originals: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/process_event.py and https://github.com/PostHog/posthog/blob/master/ee/clickhouse/models/event.py.
This file is essentially these two Python modules (combined because of the helpers' need to access `EventProcessor.db`).